src/app/core/actions/GroupAction.ts

```diff
@@ -51,6 +53,12 @@ export class GroupAction implements Action {
         return `Grouped ${this.actions.length} actions` ;
     }
 
+    public getCustomInfo(): string[] | undefined {
+        if (this.customInfo)
+            return this.customInfo;
+        return undefined;
```

Comment:
I think this `if` is redundant, you should be able to just directly return `this.customInfo` since if it is undefined, then undefined will still be returned